packages/config/src/routes.ts

```diff
@@ -1,4 +1,5 @@
 export enum AppRoute {
+  Initialise = 'initialise',
```

Comment:
added a route for this - there's one strangeness involved.. you can navigate to that URL manually at any time.
This won't work because the queries would then require authentication if the server isn't in bootstrap mode - thought to handle that later after we've refined the status queries.